tests/smoke.rs

```diff
@@ -1,13 +1,17 @@
 extern crate dlmalloc;
 extern crate rand;
 
+#[cfg(any(target_os = "linux", target_arch = "wasm32", target_os = "macos"))]
```

Comment:
I think it's ok to remove all this `#[cfg]`. While it's needed to run tests the tests aren't run on every platform under the sun.